browse-scaler/src/__tests__/cache.test.js

```diff
@@ -0,0 +1,27 @@
+const cache = require ('../cache');
+const { readFile } = require ('../util');
+
+describe ('cache tests', () => {
+    test ('data round trip', async () => {
+        const imgData = await readFile ('__tests__/stars.jpg');
+        cache.cacheImage ('someData', imgData);
```

Comment:
I'm noticing some odd looking whitespace in tests files like this. Is that a linter setting?
```suggestion
        cache.cacheImage('someData', imgData);
```

Reply:
Yes, I haven't setup my editor beyond defaults for js styles